akka-testkit-typed/src/main/scala/akka/typed/testkit/BehaviourTestkit.scala

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (C) 2014-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.testkit
```

Comment:
Should it be `akka.testkit.typed`? With persistence and friends we do `akka.persistence.typed` so I guess this should also suffix with typed?

Reply:
Yes I think that makes sense, i'll re-do